api/src/test/java/org/openmrs/api/AdministrationServiceTest.java

```diff
@@ -74,25 +69,56 @@ public void runBeforeEachTest() {
 			adminService.setImplementationIdHttpClient(implementationHttpClient);
 			cacheManager = Context.getRegisteredComponent("apiCacheManager", CacheManager.class);
 		}
-		
+
+	}
+
+	/**
+	 * @see AdministrationService#getSystemInformation()
+	 */
+	@Test
+	public void getSystemInformation_shouldReturnSystemInformation() {
+		Map<String, Map<String, String>> systemInfoMap = Context.getAdministrationService().getSystemInformation();
+		assertEquals(5, systemInfoMap.size());
+	}
+
+	/**
+	 * @see AdministrationService#getSystemInformation()
+	 */
+	@Test
+	public void getSystemInformation_shouldAddAllLoadedModulesInformation() {
+		Module module = new Module("name", "moduleId", "packageName", "author", "description", "version");
+		ModuleFactory.loadModule(module, false);
+		int numberOfModules = ModuleFactory.getLoadedModules().size();
+		Map<String, Map<String, String>> systemInfoMap = Context.getAdministrationService().getSystemInformation();
+		assertEquals(numberOfModules + 1, systemInfoMap.get("SystemInfo.title.moduleInformation").size());
```

Comment:
you could assert that the module that you loaded is in there.

at the moment you assert that the number matches what you expect but it could as well have been the same number before you loaded the module. You did not check that.